js/src/components/shipping-rate-section/estimated-shipping-rates-card/rate-form-modals/validateShippingRateGroup.js

```diff
@@ -20,6 +20,13 @@ const validate = ( values ) => {
 		);
 	}
 
+	if ( values.rate === undefined ) {
```

Comment:
💅  Similar concern as the second paragraph in https://github.com/woocommerce/google-listings-and-ads/pull/1473#discussion_r864608844.